standalone/test/include/psen_scan_v2_standalone/util/expectations.h

```diff
@@ -25,11 +25,13 @@ namespace psen_scan_v2_standalone_test
 
 #define EXPECT_BARRIER_OPENS(barrier, wait_timeout) EXPECT_TRUE(barrier.waitTillRelease(wait_timeout))
 
-#define EXPECT_DOES_NOT_BLOCK(statement)                                                                               \
+#define EXPECT_DOES_NOT_BLOCK_NOR_THROW(statement)                                                                     \
```

Comment:
I think it should either be called `EXPECT_DOES_NEITHER_BLOCK_NOR_THROW` or `EXPECT DOES_NOT_BLOCK_OR_THROW`.

Maybe its even easier to name it `EXPECT_IMMEDIATE_SUCCESSFULL_RETURN`

Reply:
Ok, thanks. What about `EXPECT_NO_BLOCK_NO_THROW`?